<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Генеалогическое дерево</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <layout class="QFormLayout" name="formLayout_9">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>ФИО:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="editFIO"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Родители:</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Отец:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="cmbFather"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Мать:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="cmbMother"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Дата рождения:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="deBirthday"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="1">
         <widget class="QCheckBox" name="cbIsAlive">
          <property name="text">
           <string>Жив</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="gbDead">
        <property name="title">
         <string>Дата смерти:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QDateEdit" name="deDead">
           <property name="time">
            <time>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
            </time>
           </property>
           <property name="maximumDate">
            <date>
             <year>9999</year>
             <month>12</month>
             <day>27</day>
            </date>
           </property>
           <property name="maximumTime">
            <time>
             <hour>23</hour>
             <minute>59</minute>
             <second>59</second>
            </time>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Пол:</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QRadioButton" name="rbMan">
           <property name="text">
            <string>муж</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbWoman">
           <property name="text">
            <string>жен</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Гражданство:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QRadioButton" name="rbRussia">
             <property name="text">
              <string>Россия</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbUkraine">
             <property name="text">
              <string>Украина</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbPoland">
             <property name="text">
              <string>Польша</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbGermany">
             <property name="text">
              <string>Германия</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbUSSR">
             <property name="text">
              <string>СССР</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_8">
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Состоял(а) на военной службе:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="cbIsMilitary">
          <property name="text">
           <string>да</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnFill">
          <property name="text">
           <string>Заполнить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAdd">
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDelete">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnCreateFile">
        <property name="text">
         <string>Создать БД</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOpenFile">
        <property name="text">
         <string>Открыть БД</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSaveFile">
        <property name="text">
         <string>Сохранить БД</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSaveAsFile">
        <property name="text">
         <string>Сохранить БД как</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actCreate"/>
    <addaction name="actOpen"/>
    <addaction name="separator"/>
    <addaction name="actSave"/>
    <addaction name="actSaveAs"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actCreate">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actSaveAs">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
